core/src/wallet/bitcoin/BitcoinLikeAccount.cpp

```diff
@@ -355,7 +355,6 @@ namespace ledger {
             abstractBlock.height = block.height;
             abstractBlock.time = block.time;
             if (BlockDatabaseHelper::putBlock(sql, abstractBlock)) {
-                BitcoinLikeBlockDatabaseHelper::putBlock(sql, block);
```

Comment:
That call disappeared and I can’t find where you moved it. What’s the logic behind that deletion?

Reply:
Because it was not doing anything:
```
bool BitcoinLikeBlockDatabaseHelper::putBlock(soci::session &sql, 
                                              const BitcoinLikeBlockchainExplorer::Block &block) {
    return false;
}
```